resources/brick/verification_datapoint_info.yml

```diff
@@ -3,192 +3,192 @@
   EnergyPlus:
     heating_setpoint:
-      point: T_heat_set
+      point: temperature_air_zone_heat_setpoint
       subject: hvac_zone
-      variable: Zone Thermostat Heating Setpoint Temperature
+      variable: Zone heating temperature setpoint
 
 ZoneCoolingSetpointMaximum:
   default:
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
       variable: cooling_setpoint
   EnergyPlus:
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
-      variable: Zone Thermostat Cooling Setpoint Temperature
+      variable: Zone cooling temperature setpoint
 
 ZoneCoolingResetDepth:
   default:
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
       variable: cooling_setpoint
   EnergyPlus:
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
-      variable: Zone Thermostat Cooling Setpoint Temperature
+      variable: Zone cooling temperature setpoint
 
 NightCycleOperation:
   default:
     zone_air_temperature_sensor:
-      point: T_zone
+      point: temperature_air_zone
       subject: hvac_zone
       variable: zone_air_temperature_sensor 
     HVAC_operation_sch:
-      point: HVAC_operation_sch
+      point: schedustatus_hvacle_hvac
       subject: Null
       variable: ahu_fan_on_off_status
     heating_setpoint:
-      point: T_heat_set
+      point: temperature_air_zone_heat_setpoint
       subject: hvac_zone
       variable: heating_setpoint
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
       variable: cooling_setpoint  
     ahu_fan_electrical_meeter:
-      point: Fan_elec_rate
+      point: power_fan
       subject: Null
       variable: ahu_fan_electrical_meeter
   EnergyPlus:
     zone_air_temperature_sensor:
-      point: T_zone
+      point: temperature_air_zone
       subject: hvac_zone
-      variable: Zone Air Temperature
+      variable: Zone air temperature
     HVAC_operation_sch:
-      point: HVAC_operation_sch
+      point: status_hvac
       subject: ahu
-      variable: Schedule Value
+      variable: HVAC operation schedule
     heating_setpoint:
-      point: T_heat_set
+      point: temperature_air_zone_heat_setpoint
       subject: hvac_zone
-      variable: Zone Thermostat Heating Setpoint Temperature
+      variable: Zone heating temperature setpoint
     cooling_setpoint:
-      point: T_cool_set
+      point: temperature_air_zone_cool_setpoint
       subject: hvac_zone
-      variable: Zone Thermostat Cooling Setpoint Temperature
+      variable: Zone cooling temperature setpoint
     ahu_fan_electrical_meeter:
-      point: Fan_elec_rate
+      point: power_fan
       subject: ahu_fan
-      variable: Fan Electric Power
+      variable: Fan power
```

Comment:
Fan Electric Power?

Reply:
I'll revisit brick testing & related- files soon. I need to remind myself.